helm/templates/ingress.yaml

```
@@ -0,0 +1,97 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "immudb.fullname" . -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+
+
+apiVersion: {{ include "immudb.chart.ingressapiversion" . }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-http
+  labels:
+    {{- include "immudb.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  ingressClassName: {{ .Values.ingress.className }}
+  {{- end }}
+  {{- if .Values.ingress.tls }}
```

I would still use additional field here such as:
```suggestion
  {{- if .Values.ingress.tls.enabled }}
```

so that you can easily disable tls (e.g. for testing) without disabling the whole tls section.

---

Good idea

---

fixed in commit e4c856577131fadfea45ea1cf822acb66c64ccfe